kodilla-spring-basic/src/main/java/com/kodilla/spring/basic/spring_dependency_injection/homework/ShippingCenter.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.spring.basic.spring_dependency_injection.homework;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShippingCenter {
+
+    @Autowired
+    @Qualifier(value = "notificationService")
+    private NotificationService notificationService;
+
+    @Autowired
```

Comment:
Wstrzykiwanie przez pola jest niewskazane i przyjelo sie robic to tylko w testach. Lepiej to robic przez kontruktor.
Mozemy do tego wrocic na spotkaniu